src/cdk/aspects/security/policies.ts

```diff
@@ -10,6 +10,13 @@ interface HasPolicyDocument extends cdk.IConstruct {
     cfnResourceType: string
 };
 
+/**
+ * The Policy class is used by Mira to validate policy aspects of various cloud services.
+ *
+ * @export
+ * @class Policies
+ * @implements {cdk.IAspect}
```

Comment:
The 3 lines above are a duplicate of the code - is there any way of extracting them automatically?

Reply:
Yes, these can be removed. I was debugging missing aspects and constructs in the documentation and added this as a test. I will remove them.